src/META-INF/plugin.xml

```diff
@@ -36,7 +36,7 @@
           <table width="100%" cellpadding="5px" border="0" style="background-color:rgb(168,75,56);">
             <tr">
               <td align="right"><b>Documentation:</b></td>
-              <td align="left"><a style="color:white;" href="http://intellij-haxe.org">Project Web Site</a></td>
+              <td align="left"><a style="color:white;" href="https://intellij-haxe.org">Project Web Site</a></td>
```

Comment:
Note to self: 
- add Discord info ?

Reply:
Just a note regarding https.  The intellij-haxe web site does not have a cert, so can't be addressed using https at this time (you will get a "this web site is not secure" screen from your browser).  I don't feel like paying $100/yr to the ISP just to protect a read-only web site that I'm not actively maintaining.  If somebody knows how to set up a cert manually that can be deployed without paying the ISP (Ionos), I'm all ears.

Reply:
you could use [letsencrypt  ](https://letsencrypt.org/)+ certbot if you  control the system and can install software

 (i personally use certbot  + [bypass ](https://www.buypass.com/products/tls-ssl-certificates/read-more-go-ssl-acme)as my provider as their certificate lasts longer before you have to renew)